Library/Homebrew/utils/curl.rb

```diff
@@ -461,6 +461,9 @@ def curl_check_http_content(url, url_type, specs: {}, user_agents: [:default], r
         url.start_with?("http://") &&
         secure_details[:final_url].present? && secure_details[:final_url].start_with?("https://")
```

Comment:
These changes need to be in another file instead. They should stop this function being called from wherever it is in the auditing code.